docs/tutorials/add-parameter-values.rst

```diff
@@ -89,10 +69,90 @@ called (must be in the same folder), with the tag ``[function]``, for example:
+| 0.000000000000000000e+00 | 4.714135898019971016e+00 |
+| 2.040816326530612082e-02 | 4.708899441575220557e+00 |
+| 4.081632653061224164e-02 | 4.702448345762175741e+00 |
+| 6.122448979591836593e-02 | 4.694558534379876136e+00 |
+| 8.163265306122448328e-02 | 4.684994372928071193e+00 |
+| 1.020408163265306006e-01 | 4.673523893805322516e+00 |
+| 1.224489795918367319e-01 | 4.659941254449398329e+00 |
+| 1.428571428571428492e-01 | 4.644096031712390271e+00 |
++--------------------------+--------------------------+
+
+Empty lines, and lines starting with ``#``, will be ignored.
+
+Then, this data should be added to the parameter file from which it will be
+called (must be in the same folder), with the tag ``[data]``, for example:
+
++---------------------+----------------------------------+--------------+-------------+
+| Name [Units]        | Value                            |  Reference   | Notes       |
++=====================+==================================+==============+=============+
+| Example [m2.s-1]    | [data]diffusivity_AuthorYear     | AuthorYear   | some data   |
++---------------------+----------------------------------+--------------+-------------+
+
+Using new parameters
+--------------------
+
+If you have added a whole new set of parameters, then you can create a new parameter set in ``pybamm/parameters/parameter_sets.py``, by just adding a new dictionary to that file, for example
+
+.. code-block:: python
 
+    AuthorYear = {
+        "chemistry": "lithium-ion",
+        "cell": "new_cell_AuthorYear",
+        "anode": "new_anode_AuthorYear",
+        "separator": "new_separator_AuthorYear",
+        "cathode": "new_cathode_AuthorYear",
+        "electrolyte": "new_electrolyte_AuthorYear",
+        "experiment": "new_experiment_AuthorYear",
+    }
+
+Then, to use these new parameters, use:
+
+.. code-block:: python
+
+    param = pybamm.ParameterValues(chemistry=pybamm.parameter_sets.AuthorYear)
+
+Note that you can re-use existing parameter subsets instead of creating new ones (for example, you could just replace "experiment": "new_experiment_AuthorYear" with "experiment": "1C_discharge_from_full_Marquis2019" in the above dictionary).
+
+It's also possible to add parameters for a single material (e.g. anode) and then re-use existing parameters for the other materials, without adding a parameter set to ``pybamm/parameters/parameter_sets.py``.
+
+.. code-block:: python
+
+    param = pybamm.ParameterValues(
+        chemistry={
+            "chemistry": "lithium-ion",
+            "cell": "kokam_Marquis2019",
+            "anode": "new_anode_chemistry_AuthorYear",
+            "separator": "separator_Marquis2019",
+            "cathode": "lico2_Marquis2019",
+            "electrolyte": "lipf6_Marquis2019",
+            "experiment": "1C_discharge_from_full_Marquis2019",
+        }
+    )
+
+or, equivalently in this case (since the only difference from the standard parameters from Marquis et al. is the set of anode parameters),
```

Comment:
could maybe refer to the "getting started" notebook for an example of doing this